tests/api.v2/routes/access.test.js

```diff
@@ -0,0 +1,45 @@
+// @ts-nocheck
+const express = require('express');
+const request = require('supertest');
+const expressLoader = require('../../../src/loaders/express');
+
+jest.mock('../../../src/api.v2/middlewares/authMiddlewares');
+jest.mock('../../../src/api.v2/controllers/accessController');
+
+describe('User access endpoint', () => {
+  let app;
+
+  beforeEach(async () => {
+    const mockApp = await expressLoader(express());
+    app = mockApp.app;
+  });
+
+  afterEach(() => {
+    jest.resetModules();
+    jest.restoreAllMocks();
+  });
+
+  it('Getting list of users to an existing experiment returns 200', async (done) => {
+    request(app)
+      .get('/v2/access/mockExperimentId')
+      .expect(200)
+      .end((err) => {
+        if (err) {
+          return done(err);
+        }
+        return done();
+      });
+  });
+
+  it('Getting list of users to an unexisting experiment returns 404', async (done) => {
+    request(app)
+      .get('/v2/access/nonExistentExperimentId')
+      .expect(404)
+      .end((err) => {
+        if (err) {
+          return done(err);
+        }
+        return done();
+      });
+  });
```

Comment:
I don't think this test is useful. Isn't it just testing the mock? I feel like the only actual functionality that is being tested here is that the request reaches the controller (which is something that is alreayd tested by the previous test too).

Reply:
This test tests that the route will pass on an error properly (the part `.catch(next)`) in the route.

Reply:
Makes sense. 

Maybe then do `it("returns errors triggered by the controller")` so it's more general, becuase if not it feels like we need to also add more tests for different kinds. of errors, when all we care about is that any error is passed on next.

Also I think the pattern of passing different ids to define what kind of error to trigger is not ideal, I think it's better to just mock the controller function to trigger an error inside this test in particular. It's more concrete about what's happening

Reply:
moved